ops/charm.py

```diff
@@ -795,16 +795,35 @@ class RelationMeta:
         role: This is :class:`RelationRole`; one of peer/requires/provides
         relation_name: Name of this relation from metadata.yaml
         interface_name: Optional definition of the interface protocol.
-        scope: "global" or "container" scope based on how the relation should be used.
+        limit: Optional definition of maximum number of connections to this relation endpoint.
+        scope: "global" (default) or "container" scope based on how the relation should be used.
     """
 
+    VALID_SCOPES = ['global', 'container']
+    DEFAULT_SCOPE = VALID_SCOPES[0]
```

Comment:
This is always dangerous. `DEFAULT_SCOPE` will be set a single time, at class evaluation time. It's not necessarily a big deal here, but every instance of the class will have the same values.